app/services/feeds/assemble_article_markdown.rb

```diff
@@ -0,0 +1,170 @@
+module Feeds
+  class AssembleArticleMarkdown
```

Comment:
this is the same as the old `RssReader::Assembler` class, only renamed `.assemble` to `.call`